lms/djangoapps/courseware/tests/test_masquerade.py

```diff
@@ -154,20 +154,6 @@ def verify_show_answer_present(self, show_answer_expected):
         self.assertIn(self.problem_display_name, problem_html)
         self.assertEqual(show_answer_expected, "Show Answer" in problem_html)
 
-    def verify_real_user_profile_link(self):
```

Comment:
IMO these tests should instead be changed to verify that the "Profile" link goes to the real user.